CHANGELOG.md

```diff
@@ -7,9 +7,32 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 -----
 
 **ENHANCEMENTS**
-
+- Add support for EFA on Centos 7, alinux and Ubuntu 1604
```

Comment:
alinux --> Amazon Linux ? --> to be aligned with others OS names

Reply:
👍